agrirouter-api-dotnet-standard-test/Service/Onboard/SecuredOnboardingServiceTestForFarmingSoftware.cs

```diff
@@ -10,13 +10,34 @@
 namespace Agrirouter.Api.Test.Service.Onboard
 {
     /// <summary>
-    /// Functional tests.
+    ///     Functional tests.
     /// </summary>
     [Collection("Integrationtest")]
     public class SecuredOnboardingServiceTestForFarmingSoftware : AbstractSecuredIntegrationTestForFarmingSoftware
     {
         private static readonly HttpClient HttpClient = HttpClientFactory.HttpClient();
 
+        [Fact]
+        public void GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAValidResponse()
```

Comment:
Wouldn't GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAnException() be more accurate?